pam_mysql.c

```diff
@@ -258,6 +258,10 @@ typedef struct _pam_mysql_ctx_t {
   int crypt_type;
   int use_323_passwd;
   int md5;
+	int sha256;
+	int sha512;
+	int blowfish;
+	int rounds;
   int sqllog;
```

Comment:
The code around uses two spaces for indentation, please don't use TAB characters in the new lines.